climada_petals/hazard/landslide.py

```diff
@@ -37,43 +37,48 @@
 LOGGER = logging.getLogger(__name__)
 HAZ_TYPE = 'LS'
 
-def sample_events_from_probs(prob_matrix, n_years, dist='binom'):
-    """sample an event set for a specified representative time span from
-    a matrix with annual occurrence probabilities
-    Draws events from chosen distribution.
+
+def sample_events(prob_matrix, n_years, dist='binom'):
+    """Repeatedly sample events"""
+    events = []
+    for i in range(n_years):
+        events.append(sample_event_from_probs(prob_matrix, n_years=1, dist=dist))
```

Comment:
Please use list comprehensions for simple for loops.
```suggestion
    events = [
        sample_event_from_probs(prob_matrix, n_years=1, dist=dist)
        for i in range(n_years)
        ]
```